Assets/Styles/index.css

```diff
@@ -0,0 +1,1062 @@
+  src: url('../fonts/Cocogoose_Pro_Italic-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose2';
+  src: url('../fonts/Cocogoose_Pro_Light_Italic-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose3';
+  src: url('../fonts/Cocogoose_Pro_Light-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose4';
+  src: url('../fonts/Cocogoose_Pro_Semilight_Italic-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose5';
+  src: url('../fonts/Cocogoose_Pro_Semilight-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose6';
+  src: url('../fonts/Cocogoose_Pro_Thin_Italic-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose7';
+  src: url('../fonts/Cocogoose_Pro_Thin-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose8';
+  src: url('../fonts/Cocogoose_Pro_UltraLight_Italic-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose9';
+  src: url('../fonts/Cocogoose_Pro_Ultralight-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Cocogoose10';
+  src: url('../fonts/Cocogoose_Pro-trial.ttf');
+}
+
+@font-face {
+  font-family: 'Lato1';
+  src: url('../fonts/Lato-Black.ttf');
+}
+
+@font-face {
+  font-family: 'Lato2';
+  src: url('../fonts/Lato-BlackItalic.ttf');
+}
+
+@font-face {
+  font-family: 'Lato3';
+  src: url('../fonts/Lato-Bold.ttf');
+}
+
+@font-face {
+  font-family: 'Lato4';
+  src: url('../fonts/Lato-BoldItalic.ttf');
+}
+
+@font-face {
+  font-family: 'Lato5';
+  src: url('../fonts/Lato-Italic.ttf');
+}
+
+@font-face {
+  font-family: 'Lato6';
+  src: url('../fonts/Lato-Light.ttf');
+}
+
+@font-face {
+  font-family: 'Lato7';
+  src: url('../fonts/Lato-LightItalic.ttf');
+}
+
+@font-face {
+  font-family: 'Lato8';
+  src: url('../fonts/Lato-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'Lato9';
+  src: url('../fonts/Lato-Thin.ttf');
+}
+
+@font-face {
+  font-family: 'Lato10';
+  src: url('../fonts/Lato-ThinItalic.ttf');
+}
+
+/* ------------ general classes -------------- */
+html,
+body {
+  height: 100%;
+  overflow: auto;
+  background-image: url('../Images/Background/mobile_background.png');
+  background-size: 100%;
+  background-repeat: no-repeat;
+  padding: 0;
+  margin: 0;
+}
+
+.line_separator {
+  border-bottom-style: solid;
+  border-width: 0.5vw;
+  border-bottom-color: #ec5242;
+  margin-left: 40.5vw;
+  width: 10vw;
+  height: 0.001vw;
+}
+
+.line_separator2 {
+  border-bottom-style: solid;
+  border-width: 0.5vw;
+  border-bottom-color: #a5a5a5;
+  margin-left: 5vw;
+  width: 90vw;
+  height: 0.001vw;
+}
+
+@keyframes blur_open {
+  0% {
+    -webkit-filter: blur(0);
+  }
+
+  100% {
+    -webkit-filter: blur(1vw);
+  }
+}
+
+@keyframes blur_close {
+  0% {
+    -webkit-filter: blur(1vw);
+  }
+
+  100% {
+    -webkit-filter: blur(0);
+  }
+}
+
+/* ------------ navigation bar -------------- */
+.desktop_navbar {
+  display: none;
+}
+
+.mobile_navbar {
+  position: fixed;
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  width: 100vw;
+  height: 18vw;
+  top: 0.001vw;
```

Comment:
* I suggest you use VW unit carefully as it takes the whole available width of your viewport you should subtract padding and margins before using this unit . This is making your page overflow kindly adjust the viewport unit.
* ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
![Screenshot (125)](https://user-images.githubusercontent.com/52497246/124195943-4b325600-dac3-11eb-917b-b6b30e00c99d.png)

Reply:
* Well done fixing this 👍